ros_ign_bridge/test/publishers/ign_publisher.cpp

```diff
@@ -1,29 +1,28 @@
-/*
- * Copyright (C) 2018 Open Source Robotics Foundation
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- *
-*/
+// Copyright 2020 Open Source Robotics Foundation, Inc.
```

Comment:
We can keep the year at 2018, since most of the file hasn't changed.

```suggestion
// Copyright 2018 Open Source Robotics Foundation, Inc.
```